service-registry/registry-service-center/src/main/java/org/apache/servicecomb/serviceregistry/RegistryUtils.java

```diff
@@ -160,6 +160,16 @@ public static MicroserviceInstance getMicroserviceInstance() {
         null : serviceCache.getInstances();
   }
 
+  // update microservice  properties
+  public static boolean updateMicroserviceProperties(Map<String, String> microserviceProperties) {
```

Comment:
RegistryUtils maybe changed in future, do not use API as possible.

Reply:
ok